src/Files/Views/SettingsPages/Experimental.xaml

```diff
@@ -93,10 +93,9 @@
                 Text="{helpers:ResourceString Name=SettingsDefaultFilesManager}" />
 
             <local:SettingsBlockControl
+                HorizontalAlignment="Stretch" IsExpanded="True"
```

Comment:
Can you fix the formatting by running XAML Styler?

Reply:
The style is changed.